jekyll/_cci2/persist-data.md

```diff
@@ -95,8 +95,8 @@ Details about individual step storage and network transfer usage can be found in
 
 ![save-cache-job-output]( {{ site.baseurl }}/assets/img/docs/job-output-save-cache.png)
 
-### How to calculate an approximation of your monthly costs
-{: #how-to-calculate-an-approximation-of-your-monthly-costs}
+### How to calculate an approximation of your storage and network costs?
```

Comment:
Just a minor question. Should we say "network and storage" or "storage and network"? Either is fine as long as try to stay consistent.

Reply:
@thomasr8 The Storage tab comes before the Network tab in the UI and the wording was reflected as such in the UI (though that changes so often so who knows if it will change again) and the page was set up to talk about Storage first, so that's why this is in this order.